resources/views/listing/partials/filter/selected.blade.php

```diff
@@ -28,61 +30,26 @@ class="flex flex-wrap gap-2 relative"
                     ].join(':')"
                 >
                     <a
-                        :href="createURL(refinement)"
+                        v-bind:href="createURL(refinement)"
                         v-on:click.prevent="item.refine(refinement)"
-                        class="flex justify-between items-center transition hover:opacity-80"
+                        class="flex items-center gap-1 p-1 text-xs rounded-lg bg transition hover:opacity-80"
                     >
-                        {{-- Why do we need an extra span here? --}}
-                        <span class="font-sans flex gap-1 p-1 items-center text-xs rounded-lg bg">
-                            {{--
-                            Having the label here is useful when filtering booleans,
-                            but the item label is currently the "code".
-                            --}}
-                            @{{ item.label }}:
-                            {{-- When the label is a boolean a yes/no would be nice --}}
+                        @{{ item.filter.name }}:
+                        <template v-if="item.filter.input === 'boolean'">
+                            <template v-if="refinement.label == '1'">@lang('Yes')</template>
+                            <template v-if="refinement.label == '0'">@lang('No')</template>
+                        </template>
```

Comment:
This will probably also happen for the swatches (i think showing the actual swatch if possible is nicer)
So maybe it's a good idea to split this up like we have done with the filters themselves: https://github.com/rapidez/core/blob/77cc7f9c116449af8dfa39b699e490264f124295/resources/views/listing/filters.blade.php#L10

Reply:
Could be done with something like this:
```blade
<template v-if="false">
@include('rapidez::listing.partials.filter.selected.boolean')
@include('rapidez::listing.partials.filter.selected.swatch')
<template v-else>
    @{{ refinement.label }}
</template>
```

We do then want the `v-else` here so we would need the `<template v-if="false">`.

Definitely cleaner if we end up having more exceptions. Would this be the nicest approach though?